### PR TITLE
[2.14] Add security.md

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: docs/security.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# Security
+*Note: This feature is available since UCRM 2.14.0-beta1
```
Comment (Piskvor) [inline docs/security.md]: Was probably supposed to be italic?

File: docs/security.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+The `public.php` file can be used to provide a customized public page or even more of them if you decide to use `$_GET` parameters with some routing.
+
+However you might want to restrict some or all of these pages to only be available to the admins of the UCRM installation and/or to the clients. Since the plugin is running on the same domain as UCRM itself, it will also receive the UCRM cookies. In this case you can use the `PHPSESSID` cookie to ask UCRM about the current session. This is done by calling the `/current-user` URL and forwarding the `PHPSESSID` cookie.
+
+You can use the received information to:
+- determine if the user can access the page
+- check if he should be somehow limited (for example that he only has read access)
```
Comment (Piskvor) [inline docs/security.md]: Users will protest gender-utral pronouns, perhaps "they"?

File: docs/security.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+- show only information relevant for that user (for example only show services for the current client)
+
+Here is an example function using CURL:
+
+```php
+function retreiveCurrentUser(string $ucrmPublicUrl): array
```
Comment (Piskvor) [inline docs/security.md]: *retrieve

File: docs/security.md (hunk 1 above)
Comment (enumag) [reply docs/security.md]: lol... @keksa broke it :-D

**Pull request merged**